GitUtils.ps1

```diff
@@ -345,17 +345,17 @@ function Stop-SshAgent() {
 }
 
 function Update-AllBranches($Upstream = 'master', [switch]$Quiet) {
-    $head = git rev-parse --abbrev-ref HEAD
-    git checkout -q $Upstream
-    $branches = (git branch --no-color --no-merged) | where { $_ -notmatch '^\* ' }
+    $head = exec { git rev-parse --abbrev-ref HEAD }
+    exec { git checkout -q $Upstream }
+    $branches = (exec { git branch --no-color --no-merged }) | where { $_ -notmatch '^\* ' }
     foreach ($line in $branches) {
         $branch = $line.SubString(2)
         if (!$Quiet) { Write-Host "Rebasing $branch onto $Upstream..." }
-        git rebase -q $Upstream $branch > $null 2> $null
+        safeexec { git rebase -q $Upstream $branch } > $null

```

Comment:
Because the original call had **2> $null**, so we should swallow all errors, that's what **safeexec** does